.github/workflows/build.yml

```diff
@@ -1,6 +1,9 @@
 name: Build
 
 on:
+  pull_request:
+    paths:
+      - .github/workflows/build.yml
```

Comment:
`tests.yml` already invokes this so this should not be needed.

Reply:
Removing this line triggered nothing though

Reply:
It did last time: https://github.com/Homebrew/homebrew-portable-ruby/pull/173/checks

Reply:
Let's give it another go